dpnp/tests/test_ndarray.py

```diff
@@ -396,12 +393,12 @@ def test_scalar_type_casting(func, shape, dtype):
 )
 @pytest.mark.parametrize("shape", [tuple(), (1,), (1, 1), (1, 1, 1)])
 @pytest.mark.parametrize(
-    "dtype", get_all_dtypes(no_float16=False, no_complex=True, no_none=True)
+    "dtype", get_all_dtypes(no_none=True, no_float16=False, no_complex=True)
```

Comment:
and probably here also?
```suggestion
    "dtype", get_all_dtypes(no_float16=False, no_complex=True)
```